### ISSUE TITLE
Define loaders for project internal files + add webcomponents polyfill

### ISSUE THREAD

**Issue by nogizhopaboroda** [holidaypirates/ascesis-storybook#9]
Add babel and other loaders only for internal files

**Comment by pankajpatel** [comment]
Is this the polyfill? [webcomponents/custom-elements](https://github.com/webcomponents/custom-elements)

**Comment by nogizhopaboroda** [comment]
yes, this one or https://github.com/webcomponents/webcomponentsjs (bundle `webcomponents-sd-ce.js`)
